### PR TITLE
Website

### LINKED ISSUE #18: developer ux: make something like https://www.multicall3.com for P256 Verifier deployment/Precompile deployment
https://www.multicall3.com

cool down the line thing

### PR DESCRIPTION
**Pull request by nalinbhardwaj**
TODO: deploy from master CI

Screenshot 2023-10-02 at 1 38 51 PM
Screenshot 2023-10-02 at 1 38 47 PM

Closes #18

### CODE REVIEW

File: website/src/app/page.tsx [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+function UsageTab() {
+  const usageCode = `bytes32 hash; // message hash
+uint256 r, s; // signature
+uint256 x, y; // public key
+
+address verifier = 0xc2b78104907F722DABAc4C69f826a522B2754De4;
+bytes memory args = abi.encode(hash, r, s, x, y);
+(bool success, bytes memory ret) = verifier.staticcall(args);
+assert(success); // never reverts, always returns 0 or 1
+bool valid = abi.decode(ret, (uint256)) == 1;`;
```
Comment (dcposch) [inline website/src/app/page.tsx]: recommend simplifying to something like

```solidity
import "p256-verifier/Verify.sol";
```
...
```solidity
bytes32 hash; // message hash
uint256 r, s; // signature
uint256 x, y; // public key

bool valid = p256VerifySignature(hash, r, s, x, y);
```

...then that function can include the malleability check

Comment (dcposch) [reply website/src/app/page.tsx]: unrelated to the website but Verify.sol could expose, say

`p256VerifySignature` and `p256VerifySignatureAllowMalleability`

**Review by dcposch — APPROVED** [review]
Looks fantastic

File: website/src/app/page.tsx (hunk 1 above)
Comment (nalinbhardwaj) [reply website/src/app/page.tsx]: good point; done here: https://github.com/daimo-eth/p256-verifier/pull/21

**Pull request merged**